e2e/e2e.sh

```diff
@@ -38,5 +40,5 @@ echo "==> installing dependencies..."
   cd "${SCRIPT_DIR}"
   echo "==> running E2E tests..."
   # shellcheck disable=SC2086
-  go test -v -timeout 1h -count 1 -tags integration ${RUN} "./..."
+  go test -mod=vendor -v -timeout 1h -count 1 -tags integration ${RUN} "./..."
```

Comment:
`-mod=vendor` is only valid when Go modules are used. We should set `GO111MODULE=on` on the invocation to be sure.